uv_threadpool_size.coffee

```diff
@@ -1,2 +1,3 @@
-process.env.UV_THREADPOOL_SIZE=16
-console.log "Set UV_THREADPOOL_SIZE=#{process.env.UV_THREADPOOL_SIZE}"
+unless process.env.UV_THREADPOOL_SIZE
```

Comment:
Will this fail weirdly if `UV_THREADPOOL_SIZE` is set to `0`?

Reply:
I'm not sure, but I can't see why anyone would be doing that?